README.md

```diff
@@ -23,3 +23,8 @@ Content to be unblocked from Cyberoam
 - **URL**: http://pwnable.kr/
 - **Blocked Under**: Games
 - **About**: 'pwnable.kr' is a non-commercial wargame site which provides various pwn challenges regarding system exploitation. This could provide a boon to security enthusiasts.
+
+
+### 5.IRC:- 
+- **About**:-Internet Relay Chat is an application layer protocol that facilitates communication in the form of text. The chat process works on a client/server networking model. IRC clients are computer programs that users can install on their system or web based applications running either locally in the browser or on 3rd party server.The freenode network can be accessed via the freenode webchat or using an IRC client such as irssi, WeeChat, ERC, HexChat, Smuxi, Quassel or mIRC. 
```

Comment:
A couple of things:

1. Please don't paste the definition of IRC, just write a few sentences about why it's important to you (us).
2. Don't you think we should have a different space for services/non-websites.
3. Are you planning to add more services?